atomics/T1098.001/T1098.001.yaml

```
@@ -0,0 +1,128 @@
+attack_technique: T1098.001
+display_name: Additional Cloud Credentials
+atomic_tests:
+- name: Azure AD Application Hijacking - Service Principal
+  auto_generated_guid: b8e747c3-bdf7-4d71-bce2-f1df2a057406
+  description: |
+    Add a certificate to an Application through its Service Principal.
+    The certificate can then be used to authenticate as the application and benefit from its rights.
+    An account with high-enough Azure AD privileges is needed, such as Global Administrator or Application Administrator. The account authentication must be without MFA.
+  supported_platforms:
+  - windows
+  input_arguments:
+    username:
+      description: Azure AD username
+      type: String
+    password:
+      description: Azure AD password
+      type: String
+    application:
+      description: Targeted Application
+      type: String
+      default: "SuperApplication"
```

```suggestion
    service_principal_name:
      description: Name of the targeted service principal
      type: String
      default: "SuperSP"
```
And please adapt in the code below, and same for 2nd test